tests/MongoDB.Bson.TestHelpers/Reflector.cs

```diff
@@ -33,5 +33,14 @@ public static object Invoke(object obj, string name)
                 .Single();
             return methodInfo.Invoke(obj, new object[] { });
         }
+
+        public static object Invoke<T1>(object obj, string name, T1 arg1)
+        {
+            var parameterTypes = new[] { typeof(T1) };
+            var methodInfo = obj.GetType().GetMethods(BindingFlags.NonPublic | BindingFlags.Instance)
+                .Where(m => m.Name == name && m.GetParameters().Select(p => p.ParameterType).SequenceEqual(parameterTypes))
+                .Single();
+            return methodInfo.Invoke(obj, new object[] { arg1 });
+        }
```

Comment:
We'll be adding more of these Invoke methods (for different number of parameters) as needed.